src/AddThreepid.ts

```diff
@@ -67,23 +68,18 @@ export default class AddThreepid {
      * @param {string} emailAddress The email address to add
      * @return {Promise} Resolves when the email has been sent. Then call checkEmailLinkClicked().
      */
-    public addEmailAddress(emailAddress: string): Promise<IRequestTokenResponse> {
-        return MatrixClientPeg.get()
-            .requestAdd3pidEmailToken(emailAddress, this.clientSecret, 1)
-            .then(
-                (res) => {
-                    this.sessionId = res.sid;
-                    return res;
-                },
-                function (err) {
-                    if (err.errcode === "M_THREEPID_IN_USE") {
-                        err.message = _t("This email address is already in use");
-                    } else if (err.httpStatus) {
```

Comment:
This bit of logging seems to get lost, is that intended?

Reply:
Yes, see https://github.com/matrix-org/matrix-react-sdk/pull/10432#discussion_r1156468143

> I don't think we need to re-label a `HTTPError` as they already have a good message that looks like `Server returned ${response.status} error: ${body}` and the main part of the error isn't translated regardless (only the word "Status" 😄)
> 
> Probably best to just remove in favor of thinking about a translations in general from the `matrix-js-sdk`, see https://github.com/matrix-org/matrix-js-sdk/issues/1309